src/python/pants/backend/python/lint/flake8/subsystem.py

```diff
@@ -94,6 +95,11 @@ class Flake8(PythonToolBase):
             """
         ),
     )
+    extra_files = FileListOption(
+        default=None,
+        advanced=True,
+        help="Path to extra YAML config files used by flake8 plugins like `flake8-bandit`.",
```

Comment:
Note that the files can be whatever you want, they don't have to be YAML config files. Maybe something like this?

> Paths to extra files to include in the sandbox. This can be useful for Flake8 plugins, like including config files for the `flake8-bandit` plugin.

Because this won't fit on one line, we use the helper function `softwrap`. You'll find examples of that in this file.